starters/basic-starter/components/media--image.tsx

```diff
@@ -17,26 +18,53 @@ export function MediaImage({
   height,
   priority,
   sizes,
+  imageStyle,
   ...props
 }: MediaImageProps) {
   const image = media?.image;
 
   if (!image) {
     return null;
   }
+  let sizeProps;
+  let srcURL;
 
-  const sizeProps =
-    layout === 'fill'
-      ? null
-      : {
-          width: width || image.resourceIdObjMeta.width,
-          height: height || image.resourceIdObjMeta.height,
+  // Use the image style to render an image if specified.
+  if (imageStyle) {
+    if (image.links) {
+      if (image.links[imageStyle]) {
+        const imageStyleSource = image.links[imageStyle];
+        sizeProps = {
+          width: width || imageStyleSource.meta.width,
+          height: height || imageStyleSource.meta.height,
         };
+        srcURL = imageStyleSource.href;
+      } else {
+        console.warn(
```

Comment:
Can we do this so that the image without an image style would be rendered when a warning is present?